content/en/docs/setup/production-environment/windows/intro-windows-in-kubernetes.md

```diff
@@ -516,26 +517,34 @@ work between Windows and Linux:
   node. They should be applied to all containers as a best practice if the operator
   wants to avoid overprovisioning entirely.
 * `securityContext.allowPrivilegeEscalation` -
-   not possible on Windows; none of the capabilities are hooked up
+   not possible on Windows; none of the capabilities are hooked up. Pod API
+   validation would fail when `IdentifyPodOS` featuregate is enabled.
 * `securityContext.capabilities` -
-   POSIX capabilities are not implemented on Windows
+   POSIX capabilities are not implemented on Windows. Pod API
+   validation would fail when `IdentifyPodOS` featuregate is enabled.
 * `securityContext.privileged` -
-   Windows doesn't support privileged containers
+   Windows doesn't support privileged containers. Pod API
+   validation would fail when `IdentifyPodOS` featuregate is enabled.
 * `securityContext.procMount` -
-   Windows doesn't have a `/proc` filesystem
+   Windows doesn't have a `/proc` filesystem. Pod API
+   validation would fail when `IdentifyPodOS` featuregate is enabled.
 * `securityContext.readOnlyRootFilesystem` -
    not possible on Windows; write access is required for registry & system
-   processes to run inside the container
-* `EcurityContext.runAsGroup` -
-   not possible on Windows as there is no GID support
-* `ecurityContext.runAsNonRoot` -
+   processes to run inside the container. Pod API
+   validation would fail when `IdentifyPodOS` featuregate is enabled.
+* `securityContext.runAsGroup` -
+   not possible on Windows as there is no GID support. Pod API
+   validation would fail when `IdentifyPodOS` featuregate is enabled.
+* `securityContext.runAsNonRoot` -
    Windows does not have a root user. The closest equivalent is `ContainerAdministrator`
-   which is an identity that doesn't exist on the node.
+   which is an identity that doesn't exist on the node. 
```

Comment:
Surprised this isn't updated against this base: https://github.com/kubernetes/website/pull/30256/files